### ISSUE TITLE
Input field - label issue

### ISSUE THREAD

**Issue by EricL2017** [creativetimofficial/material-dashboard-pro#45]
Hi Axel

I have a form that I populate with information from a DB, using JS/Ajax. When I change the value of the input field, the label remains in the "empty position" and shows on top of the information in the field. See below the image. Also below is the HTML and JS code.

Thanks for your help, as usual.

Eric
[HTML]
<div class="col-md-5">
    <div class="form-group label-floating">
        <label class="control-label">First name</label>
        <input type="text" class="form-control" name="user_firstname" id="user_firstname">
    </div>
</div>

[JS]
document.getElementById("user_firstname").value = "something"

<img width="263" alt="screen shot 2017-06-04 at 11 12 05 pm" src="https://cloud.githubusercontent.com/assets/28696680/26765385/6cc1cd1a-497b-11e7-9ed8-3e6425d2d693.png">

**Comment by street1030** [comment]
I'm also having this issue.  Any updates on this?

**Comment by street1030** [comment]
Make sure you don't have the .is-empty class on the parent by default.  This is added by the javascript and shouldn't exist by default.

**Comment by EricL2017** [comment]
Hi Street1030. Thanks for your input. Can you please elaborate and explain how this can be fixed?

**Comment by alexandru-paduraru** [comment]
@EricL2017 thank you for pointing out. The solution, as @street1030 pointed out, would be, after you make the Ajax call, on the response of it, to remove the class `.is-empty` from the parent `div.form-group` of the input. In jQuery this can be done like this `$('#your-input-selector-here').parent().removeClass("is-empty");`

Hope this helps you.

**Comment by EricL2017** [comment]
Hi Alex, thanks for the explanation and guidance to fix this issue. It works perfectly fine now. Great support!
